call-home/src/transmitter.rs

```diff
@@ -0,0 +1,9 @@
+/// 'client' module contains type(s) which help
```

Comment:
why are these lines so "short" ?

Reply:
I thought it looked better this way 😓 .

Reply:
Use use 100 in the control-plane and by copy I guess here too: https://github.com/openebs/mayastor-extensions/blob/815005f3b27d4cb719b22d266e35ca3104dd6e8c/.rustfmt.toml#L1